reinforcement_learning/common/sagemaker_rl/ray_launcher.py

```diff
@@ -234,14 +234,18 @@ def create_tf_serving_model(self, algorithm=None, env_string=None):
         cls = get_agent_class(algorithm)
         with open(os.path.join(MODEL_OUTPUT_DIR, "params.json")) as config_json:
             config = json.load(config_json)
-        print("Loaded config for TensorFlow serving.")
-        config["monitor"] = False
-        config["num_workers"] = 1
-        config["num_gpus"] = 0
-        agent = cls(env=env_string, config=config)
-        checkpoint = os.path.join(MODEL_OUTPUT_DIR, "checkpoint")
-        agent.restore(checkpoint)
-        export_tf_serving(agent, MODEL_OUTPUT_DIR)
+        use_torch = config.get("use_pytorch", False)
```

Comment:
Can we have a separate function `create_pytorch_serving_model`? It make more sense to have the branch on a higher level, e.g. Torch -> create_torch_serving_model, TF->create_tf_serving_model

Reply:
Refactored, thanks!